src/Mover.php

```diff
@@ -39,23 +40,17 @@ public function movePackage(Package $package)
         $filesystem = new Filesystem(new Local($this->workingDir));
 
         foreach( $package->autoloaders as $autoloader ) {
-            foreach( $autoloader->paths as $path ) {
+            foreach ($autoloader->paths as $path) {
                 $source_path = $this->workingDir . '/vendor/' . $package->config->name . '/' . $path;
 
                 $finder->files()->in($source_path);
 
-                $searchNamespace = $autoloader->namespace;
-
-                if ( is_a($autoloader, Psr4::class)) {
-                    $searchNamespace = trim($autoloader->namespace, '\\');
-                }
-
                 foreach ($finder as $file) {
-                    if ( is_a($autoloader, Psr0::class)) {
-                        $targetFile = str_replace($this->workingDir, $this->config->dep_directory, $file->getRealPath());
-                    } else {
-                        $namespacePath = str_replace('\\', '/', $autoloader->namespace);
+                    if (is_a($autoloader, NamespaceAutoloader::class)) {
```

Comment:
You should prefer `instanceof` operator instead of `is_a()` function. The operator doesn't incur a function overhead, and `is_a()` was only de-deprecated because of popular request. It should have been gone already.